spec/controllers/quotes_controller_spec.rb

```diff
@@ -12,19 +29,53 @@
         get :index
         expect(assigns(:customer_requests)).to eq([cr1, cr2, cr3])
       end
+      it 'assigns all the customers open quotes to @open_quotes' do
+        company1 = create(:company)
+        company2 = create(:company)
+        customer = create(:customer)
+        sign_in customer
+        cr1 = create(:customer_request, customer_id: customer.id)
+        cr2 = create(:customer_request, customer_id: customer.id)
+        cr3 = create(:customer_request, customer_id: customer.id)
+        q1 = create(:quote,
+          company_id: company1.id,
+          customer_request_id: cr1.id,
+          accepted: nil
+          )
+        q2 = create(:quote,
+          company_id: company1.id,
+          customer_request_id: cr2.id,
+          accepted: true
+          )
+        q3 = create(:quote,
+          company_id: company1.id,
+          customer_request_id: cr3.id,
+          accepted: nil
+          )
+        q4 = create(:quote,
+          company_id: company2.id,
+          customer_request_id: cr1.id,
+          accepted: true
+          )
+        q5 = create(:quote,
+          company_id: company2.id,
+          customer_request_id: cr2.id,
+          accepted: nil
+          )
+        get :index
+        expect(assigns(:open_quotes)).to include(q1, q3, q5)
```

Comment:
I realize this is problematic unless we also infer that it doesn't have the quotes we don't want. So as part of the test you can have a second expect:

```ruby
expect(assigns(:open_quotes)).to_not include(q2, q4)
```